DEVELOPMENT.md

```diff
@@ -195,7 +195,10 @@ for your `KO_DOCKER_REPO` if required. To be able to push images to
 ```shell
 gcloud auth configure-docker
 ```
-The [example GKE setup](#using-gke) in this guide grants service accounts permissions to push and pull GCR images in the same project.
+
+To be able to pull images from `gcr.io/<project>`, please follow the instructions [here](https://cloud.google.com/container-registry/docs/access-control#grant) to configure IAM policies for the services that will pull iamges from your GCR. A list of GCP default service accounts can be found [here](https://cloud.google.com/iam/docs/service-accounts#types)
```

Comment:
`GCP default service accounts` -> `GCP Computer Engine default service accounts`. GKE are actually clusters of compute engines (https://cloud.google.com/compute/docs/access/service-accounts#default_service_account)